.github/CODEOWNERS

```diff
@@ -50,7 +50,7 @@
 /tests/modules/site-health/audit-enqueued-assets                            @manuelRod
 /tests/testdata/modules/site-health/audit-enqueued-assets                   @manuelRod
 
-# Module: Persistent Object Cache Health Check
+# Module: Persistent Object Cache
```

Comment:
The names here are all based on the actual module name. If we change this here, we should probably rename the module itself as well (in `modules/object-cache/persistent-object-cache-health-check/load.php`).